po/LINGUAS

```diff
@@ -69,3 +69,5 @@ zh_HK
 zh_TW
 oc
 et
+ast
+fur
```

Comment:
Nitpick, but maybe we could have it sorted alphabetically?

Reply:
Right, fixed.